plugins/woocommerce-admin/client/tasks/fills/PaymentGatewaySuggestions/index.js

```diff
@@ -185,6 +191,37 @@ export const PaymentGatewaySuggestions = ( { onComplete, query } ) => {
 		return gateway;
 	}, [ isResolving, query, paymentGateways ] );
 
+	const isWCPayOrOtherCategoryDoneSetup = useMemo( () => {
+		for ( const [ , gateway ] of paymentGateways.entries() ) {
+			if ( ! gateway.installed || gateway.needsSetup ) {
+				continue;
+			}
+
+			if (
+				gateway.plugins?.length === 1 &&
+				gateway.plugins[ 0 ] === 'woocommerce-payments'
+			) {
+				return true;
+			}
+
+			if (
+				gateway.category_other &&
+				gateway.category_other.indexOf( countryCode ) !== -1
```

Comment:
Looks like we use these condition checks in many areas. How about creating some helpful functions for them?

```
gateway.plugins?.length === 1 &&
gateway.plugins[ 0 ] === 'woocommerce-payments'

gateway.category_other &&
gateway.category_other.indexOf( countryCode ) !== -1

gateway.plugins?.length === 1 &&
gateway.plugins[ 0 ] === 'woocommerce-payments'			
```

Reply:
Looks nice. Thanks for updating them. 🙌